lib/percona_migrator/lhm_parser.rb

```diff
@@ -1,53 +0,0 @@
-module PerconaMigrator
```

Comment:
We still have to see how both, LHM and regular migrations DSL, fit together but I'd like to implement it similarly: make LHM translate the the DSL to MySQL's DDL and then execute it through Percona.